templates/home/index.html.twig

```diff
@@ -1,7 +1,81 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}NR Symfony homework{% endblock %}
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% for index, team in json %}
+            {% for student in team["students"] %}
+                <li class="list-group-item">
+                    <a href="{{ path('student', {'name': student|url_encode , 'project': index|url_encode }) }}">{{student}}</a>
```

Comment:
`path` jau viduje padaro `url_encode`.

Pabandyk be jo – turbūt dėl to ir testai neprasisuka